predicators/ground_truth_models/kitchen/options.py

```diff
@@ -0,0 +1,101 @@
+"""Ground-truth options for the Kitchen environment."""
+
+from typing import Dict, Sequence, Set
+
+import numpy as np
+from gym.spaces import Box
+
+try:
+    from mujoco_kitchen.utils import \
+        primitive_and_params_to_primitive_action, primitive_idx_to_name, \
+        primitive_name_to_action_idx
+except ImportError:
+    pass
```

Comment:
instead of passing here, you probably want to set some variable like `kitchen_env_imported = False`. Then, you should `assert kitchen_env_imported` in `get_options` or `__init__` or somewhere

Reply:
Does not work because CFG gets populated later.

Reply:
I think @nkumar-bdai was just suggesting a global variable, and that goes along with my other comment